requirements.txt

```diff
@@ -1,12 +1,14 @@
 cachetools >= 2.0.0
-dlib >= 19.16.0
+dlib == 19.16.0
+ipdb == 0.12
 librosa >= 0.6.2
 pandas >= 0.18.0
 pyannote.core >= 3.0
 pyannote.database >= 2.2
 pyannote.generators >= 2.0
 pyannote.metrics >= 1.8.1
 pyannote.pipeline >= 1.2
+pyannote.db.musan == 0.1.2
```

Comment:
I don't think we should instead `MUSAN` at the same time as `pyannote.audio`, should we?